packages/manager/modules/pci/src/projects/project/storages/cold-archive/add/translations/Messages_fr_FR.json

```diff
@@ -11,8 +11,8 @@
   "pci_projects_project_storages_cold_archive_add_action_create": "Créer un conteneur",
   "pci_projects_project_storages_cold_archive_add_action_cancel": "Annuler",
   "pci_projects_project_storages_cold_archive_add_action_create_pending": "Conteneur en cours de création",
-  "pci_projects_project_storages_cold_archive_add_action_create_archive_create_request_success_linked_user": "Le conteneur {{container}} a été créé avec succès. L'utilisateur S3 {{ userName }} est administrateur de ce conteneur.",
+  "pci_projects_project_storages_cold_archive_add_action_create_archive_create_request_success_linked_user": "Le conteneur {{container}} a été créé avec succès et l'utilisateur S3 {{ userName }} a été ajouté en tant que Administrator.",
   "pci_projects_project_storages_cold_archive_add_action_create_archive_create_request_success_no_linked_user": "Le conteneur {{container}} a été créé.",
-  "pci_projects_project_storages_cold_archive_add_action_create_archive_create_request_success": "Le conteneur {{containerName}} a été créé avec succès. L'utilisateur S3 {{ userName }} est administrateur de ce conteneur.",
+  "pci_projects_project_storages_cold_archive_add_action_create_archive_create_request_success": "Le conteneur {{containerName}} a été créé avec succès et l'utilisateur S3 {{ userName }} a été ajouté en tant que Administrator.",
```

Comment:
```suggestion
  "pci_projects_project_storages_cold_archive_add_action_create_archive_create_request_success": "Le conteneur {{containerName}} a été créé avec succès et l'utilisateur S3 {{ userName }} a été ajouté en tant qu'Administrateur.",
```

Reply:
j'ai utilisé la nomenclature utilisée dans la spécification du ticket.

Reply:
i guess we could challenge the PU.
if they are not agree let me know.